evohomeclient2/hotwater.py

```diff
@@ -35,33 +32,47 @@ def _set_dhw(self, data):
         response.raise_for_status()
 
     def set_dhw_on(self, until=None):
-        """Set the DHW on until a given time, or permanently."""
+        """Sets the DHW on until a given time, or permanently."""
         if until is None:
-            data = {"Mode": "PermanentOverride", "State": "On", "UntilTime": None}
+            data = {"Mode": "PermanentOverride",
+                    "State": "On",
+                    "UntilTime": None}
         else:
-            data = {
-                "Mode": "TemporaryOverride",
-                "State": "On",
-                "UntilTime": until.strftime("%Y-%m-%dT%H:%M:%SZ"),
-            }
+            data = {"Mode": "TemporaryOverride",
+                    "State": "On",
+                    "UntilTime": until.strftime('%Y-%m-%dT%H:%M:%SZ')}
 
         self._set_dhw(data)
 
     def set_dhw_off(self, until=None):
-        """Set the DHW off until a given time, or permanently."""
+        """Sets the DHW off until a given time, or permanently."""
         if until is None:
-            data = {"Mode": "PermanentOverride", "State": "Off", "UntilTime": None}
+            data = {"Mode": "PermanentOverride",
+                    "State": "Off",
+                    "UntilTime": None}
         else:
-            data = {
-                "Mode": "TemporaryOverride",
-                "State": "Off",
-                "UntilTime": until.strftime("%Y-%m-%dT%H:%M:%SZ"),
-            }
+            data = {"Mode": "TemporaryOverride",
+                    "State": "Off",
+                    "UntilTime": until.strftime('%Y-%m-%dT%H:%M:%SZ')}
 
         self._set_dhw(data)
 
     def set_dhw_auto(self):
-        """Set the DHW to follow the schedule."""
-        data = {"Mode": "FollowSchedule", "State": "", "UntilTime": None}
+        """Sets the DHW to follow the schedule."""
+        data = {"Mode": "FollowSchedule",
+                "State": "",
+                "UntilTime": None}
 
         self._set_dhw(data)
+
+
+    def get_dhw(self):
```

Comment:
I think should be called:
 - either `get_dhw_state()` to be consistent with `set_dhw_auto()` and `set_dhw_on()`, 
 - or maybe `get_state()`, given that its a method of the **HotWater** object class

Reply:
I agree it is more logical.